_pages/index.md

```diff
@@ -3,16 +3,38 @@
+Neuroimaging experiments result in complicated data that can be arranged in many different ways.
+So far there is no consensus how to organize and share data obtained in neuroimaging experiments.
+Even two researchers working in the same lab can opt to arrange their data in a different way.
+Lack of consensus (or a standard) leads to misunderstandings and time wasted on rearranging data or rewriting scripts expecting certain structure.
+With the Brain  Imaging Data Structure (BIDS), we describe a simple and easy to adopt way of organizing neuroimaging and behavioral data.
 
-![](./assets/img/dicom-reorganization-transparent-white_1000x477.png){: .center-image }
+![BIDS-folder-organization](./assets/img/dicom-reorganization-transparent-white_1000x477.png){: .center-image }
 
-BIDS is heavily inspired by the format used internally by the OpenfMRI repository (now known as [OpenNeuro.org](http://openneuro.org)). While working on BIDS we consulted many neuroscientists to make sure it covers most common experiments, but at the same time is intuitive and easy to adopt. The specification is intentionally based on simple file formats and folder structures to reflect current lab practices and make it accessible to a wide range of scientists coming from different backgrounds.
+BIDS is heavily inspired by the format used internally by the OpenfMRI repository (now known as [OpenNeuro.org](http://openneuro.org)).
+While working on BIDS we consulted many neuroscientists to make sure it covers most common experiments, but at the same time is intuitive and easy to adopt.
+The specification is intentionally based on simple file formats and folder structures to reflect current lab practices and make it accessible to a wide range of scientists coming from different backgrounds.
 
-Find a good introduction to the BIDS standard in the [paper published in Nature Scientific Data](http://www.nature.com/articles/sdata201644).
+## BIDS is a community effort
 
-Look through some of the community's [presentations on BIDS](https://osf.io/yn93h/).
+BIDS is developed by the community for the community and everybody can [become a part of the community](https://bids.neuroimaging.io/get_involved.html).
 
-Take a look at how the community [uses BIDS](https://medium.com/stanford-center-for-reproducible-neuroscience/bids-usage-survey-results-72637ff039c4).
+## Specification vs. Ecosystem
 
-Sign up to receive occasional [updates by email](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit) and follow BIDS on [Twitter](https://twitter.com/BIDSstandard?ref_src=twsrc%5Etfw).
+Since the inception of BIDS as a specification on how to organize neuroimaging data, a large ecosystem of tools and ressources has evolved around BIDS.
```

Comment:
"Since the inception of the BIDS specification that documents how to organize neuroimaging data, ..."